ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/router-bgp.md

```diff
@@ -1442,6 +1467,13 @@
             # Outbound RCF function name with parenthesis.
             # Example: MyFunction(myarg).
             rcf_out: <str>
+            default_route:
```

Comment:
Can we add the same for EVPN neighbors. Here we did for EVPN peer group

Reply:
Done